AT-1469 remove log configuration and add example for debug (#36)
Bit-Quill/amazon-timestream-driver-jdbc: README.md

### Building and using the Javadoc JAR to extract the Javadoc HTML files (amazon-timestream-jdbc-\<version\>-javadoc.jar)
Javadoc JAR builds with `mvn install` alongside the other JAR files. To extract the Javadoc HTML files, use the following command: `jar -xvf amazon-timestream-jdbc-1.0.0-javadoc.jar`

### Debug with BI tools
#### DBeaver

> To enable debug logs, the application should be run with `-Ddbeaver.trace.enabled=true` flag

Reviewer: Let's add more examples, similar to what DBeaver suggests - https://github.com/dbeaver/dbeaver/wiki/JDBC-Tracing

As an example - where to enable it, complete instructions for the user (e.g. restart DBeaver), where to find log files, the name of log files, and an example of the log generated by the driver.

#### DbVisualizer
To enable debug mode, the following steps should be done:
1. Open `Tools -> Debug Window`
2. Open the `Debug Log` tab

> 3. Enable the `Debug DbVisualizer` checkbox

Reviewer: Add instructions on where to view log files, and also an example of the actual log generated by the driver.

#### Tableau Desktop
To enable debug mode need to run Tableau Desktop with -DLogLevel=debug flag
More details on the official [documentation](https://kb.tableau.com/articles/howto/how-to-create-tableau-desktop-debug-logs)

#### Java Application

> Need to change log level to Debug/FINE. See official documentation how to change log level.

Reviewer: ```suggestion
Need to change the log level to `DEBUG` or `FINE`. 
```

> 

Reviewer: Need to add an example/code sample on how to configure it.
